ts/Dashboards/EditMode/SidebarPopup.ts

```diff
@@ -513,7 +518,11 @@ class SidebarPopup extends BaseForm {
 
         const componentPromise =
             Bindings.addComponent(options, sidebar.editMode.board, newCell);
-        sidebar.editMode.setEditOverlay();
+
+        sidebar.editMode.setEditOverlay(
+            undefined,
+            !this.editMode.options.settings?.enabled
+        );
```

Comment:
```suggestion

        sidebar.editMode.setEditOverlay(
            true
        );
```

would this not have worked? And then we would not have to add `forceRemove`?